client/landing/gutenboarding/onboarding-block/style-preview/style.scss

```diff
@@ -54,10 +58,10 @@
 	&:hover {
 		box-shadow: inset 0 0 0 1px var( --studio-gray-90 );
 	}
+}
 
-	span {
-		align-self: baseline;
-	}
```

Comment:
I think deleting this CSS is the only change you need to do.

Reply:
We had that initially. It's pretty good, but if we tweak things to be really big with underlines, we can see the issue this addressed initially where fonts with different vertical heights aren't baseline aligned 🤢 

![Screen Shot 2020-03-31 at 17 28 22](https://user-images.githubusercontent.com/841763/78044491-0d50fa00-7375-11ea-9113-e65e8e892ebc.png)

Compare with this branch ✨ 

![Screen Shot 2020-03-31 at 17 30 12](https://user-images.githubusercontent.com/841763/78044688-4db07800-7375-11ea-8f83-cd5fa56ad17f.png)